src/main/java/ch/uzh/ifi/hase/soprafs22/controller/GameController.java

```diff
@@ -252,4 +252,23 @@ public void submitWhiteCard(@RequestHeader(value = "authorization", required = f
         userService.deleteWhiteCard(id, whiteCard);
       }
     }
+
+  /**
+   * Gets the current black card of a user or throws status exception
+   * @param token: token of the user
+   * @param userId: userId of the user
+   * @return: CardGetDTO
+   */
+    @GetMapping("/users/{userId}/blackCard")
```

Comment:
i know in our application we only get at most 1 card, but i think we should still call it "blackCards" (as we did in the rest interface) to adhere to the standards mentioned in software engineering. What do you think? I'm fine to keep it in singular, but then we should adapt the rest interface, to reflect the design decision